trade-finance-manager-ui/server/api-ssrf.test.js

```diff
@@ -3,6 +3,9 @@ const MockAdapter = require('axios-mock-adapter');
 const api = require('./api');
 
 const mockAxios = new MockAdapter(axios);
+const localIp = '127.0.0.1';
+const urlTraversal = '../../../etc/stealpassword';
+const validId = '5ce819935e539c343f141ece';
```

Comment:
I've added a test to the bottom of this file, but also done a bit of a refactor to reduce some DRY content (moved these variables out up here, and commonised some more of the amendments tests)